OneCommunityGlobal/HighestGoodNetworkApp
# Zijie lint fix for reports component Phase 5

# Description
Lint fix for Reports component, this is phase 5.

## Files changed
All js/jsx file in /src/components/Reports/TeamReport

eslint test:
<img width="1432" alt="CleanShot 2023-09-30 at 17 32 55@2x" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/46791134/a8846c30-c0cb-46e6-b2be-6d5edca7397f">


# How to Test

To test this pull request:

- Check out the current branch
- Run `npm install` and `npm run start:local` to run this PR locally  
- Log in as an admin user
- Check all the functionality of the report is the same as before, and see if any bugs have appeared. (Please record a video)

(Optional)
- Set up ESLint in VSCode/ IntelliJ IDEA, I personally use IntelliJ IDEA
- Delete .eslintignore file in the root: `rm .eslintignore`
- `npx eslint src/components/Reports/TeamReport/*.{js,jsx}`
- `git restore .`

I have complete confidence that **the code will pass the ESLint tests.** Therefore, I kindly request that you **focus on testing the frontend**. Specifically, please ensure that the **report functionality remains consistent** with how it operated previously. Thank you for your time and effort!
